sharaf-snunit/src/ba/sake/sharaf/snunit/SnunitSharafRequest.scala

```diff
@@ -16,8 +16,22 @@ class SnunitSharafRequest(underlyingRequest: SnunitRequest) extends Request {
         HttpString(headerName) -> Seq(headerValue)
       }
 
-  def cookies: Seq[Cookie] = ??? // TODO
-  // underlyingHttpServerExchange.requestCookies().asScala.map(CookieUtils.fromUndertow).toSeq
+  def cookies: Seq[Cookie] =
+    val builder = Seq.newBuilder[Cookie]
+    val underlyingHeaders = underlyingRequest.headers
+    // TODO: Use underlyingRequest.cookieFieldIndex when available
+    underlyingHeaders.foreach {
+      case ("Cookie", cookieString) =>
+        cookieString.split('=').foreach {
```

Comment:
Should probably split by a semicolon `;` first, and then each entry by `=`, in case of multiple cookies?  
This would only handle the case where only one is sent.

Reply:
Sorry, it was a typo. I wanted to split by `;` as you said.